#! /bin/sh /usr/share/dpatch/dpatch-run
## pdftops-testsuite.dpatch by Martin Pitt <mpitt@debian.org>
##
## DP: Fix path to pdftops in the test suite.
## DP: http://www.cups.org/str.php?3161

@DPATCH@
diff -urNad cups-1.3.9~/test/run-stp-tests.sh cups-1.3.9/test/run-stp-tests.sh
--- cups-1.3.9~/test/run-stp-tests.sh	2008-07-14 20:29:58.000000000 +0200
+++ cups-1.3.9/test/run-stp-tests.sh	2009-01-29 07:23:05.000000000 +0100
@@ -277,7 +277,7 @@
 else
 	ln -s $root/filter/imagetops /tmp/cups-$user/bin/filter
 	ln -s $root/filter/imagetoraster /tmp/cups-$user/bin/filter
-	ln -s $root/pdftops/pdftops /tmp/cups-$user/bin/filter
+	ln -s $root/filter/pdftops /tmp/cups-$user/bin/filter
 fi
 
 #
